src/azure-cli-core/azure/cli/core/error_assistance.py

```diff
@@ -0,0 +1,172 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+import shutil
+import requests
+
+from typing import Union
+from azure.cli.core import AzCli
+from azure.cli.core.style import Style, print_styled_text
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+_DEEPPROMPT_ENDPOINT = "https://data-ai-dev.microsoft.com/deepprompt/api/v1"
+_DEEPPROMPT_APP = "7d78b7a3-e228-4b85-8fcf-5633fb326beb"
+_AAD_TENANT = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+_SCOPES = [f"{_DEEPPROMPT_APP}/.default"]
```

Comment:
I would like to confirm if there is any risk in exposing this information in open-source project? Are they all publicly available?